catalog-info.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elasticsearch-k8s-metrics-adapter
+  description: Buildkite Pipeline for elasticsearch-k8s-metrics-adapter
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/elasticsearch-k8s-metrics-adapter
+
+spec:
+  type: buildkite-pipeline
+  owner: group:cloud-k8s-operator
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: elasticsearch-k8s-metrics-adapter
+    spec:
+      repository: elastic/elasticsearch-k8s-metrics-adapter
```

Comment:
FYI defaults values for the `provider_settings` are defined [here](https://github.com/elastic/ci/blob/685b84f1545ac68c3553a1316d5ef2da4a8cc1a3/terrazzo/terrazzo/config/defaults/buildkite/pipelines.py#L4), which should be ok.

You can look at the settings of another serverless project via the ui to compare: https://buildkite.com/elastic/elasticsearch-controller/settings/repository.

Reply:
I think the defaults look fine for now, will tweak in the next PR as we add actual things to the pipeline